.changeset/gorgeous-clocks-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'eslint-config-seek': major
+---
+
+Error on unused React imports
```

Comment:
Thoughts on https://github.com/seek-oss/eslint-config-seek/pull/145/commits/e1daf07296a785cf1c55899338c4693abb94eb36 as an alternate take on this?

Reply:
If memory serves, we used to have this rule, but removed it because your IDE autofixing-away unused imports on save was annoying for lots of users (myself included). Happy to take this to a wider audience though.

Reply:
Have rolled back the React changes - we probably don't need to muddy the waters in this version

Reply:
I honestly don't mind the react change, though we _could_ save it for when drop react 18 support and _maybe_ we could try out https://www.npmjs.com/package/eslint-plugin-react-x at the same time.

Reply:
:+1: I think bundling could make sense, I'm happy to leave it out for now 🙏